SPEC.md

```diff
@@ -323,6 +328,20 @@ An example trace:
 
 ```
 
+### Measurements
+
+Measurements are a separate kind of document that engines can emit. Measurements
+are a numeric value to a given named key. Measurements are primarily used for
+tracking rates of certain kinds of errors engines may encounter internally (e.g.
+parse errors).
+
+```json
+{
+  "type": "measurement",
+  "name": "foo.bar",
+  "value": 42
```

Comment:
Are we limiting value to integers or will we allow floats as well?

Reply:
Given the conversation with Bryan & the potential larger future purposes for this document type, I'm not sure it's reasonable to limit ourselves to ints at the spec level.

Reply:
Sounds good. Just wasn't clear from the spec or example if floats were allowed as well.